src/Nri/Ui/SegmentedControl/V8.elm

```diff
@@ -56,6 +56,19 @@ type alias ToggleConfig a msg =
     }
 
 
+{-| Same shape as ToggleConfig but with an optional selected. This would ideally
+be the same as ToggleConfig but we as Zambonis don't have time in the ticket to
+also upgrade all existing uses of viewToggle. Katie is mentally noting this as a
+good hackday clean up but if you find it and fix it first, that's great too!
+-}
```

Comment:
💭 I think it actually makes sense to have two different configs, because in the case of "there's always a selection" the selected shouldn't be a `Maybe`.

Reply:
I don't want to change the config for the nav element because there _should_ always be a selection. But for the element that acts kind of like a select, we should think of it as a radio button collection--it's possible to not have a default! But it's easy for the end user to provide a default by passing `Just blah`.